### PR TITLE
brand new SectionTitle !

### PR DESCRIPTION
**Pull request by vbouillennec**
Yo @adrien-nf  tu peux tester de merger ça ? ça ne va rien casser tkt fréro

### CODE REVIEW

File: src/components/Common/Layout/Layout.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default function Layout({ children }: LayoutSpec) : JSX.Element {
             items-center
         ">
             <div className="">
+				{/* <Rectangle/> */}
```
Comment (adrien-nf) [inline src/components/Common/Layout/Layout.tsx:19]: What does the comment mean? Why does the upper line have an empty classname?

File: src/components/Common/SectionTitle/SectionTitle.spec.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { ReactNode } from "react";
+
+export interface SectionTitleSpec {
+    children: ReactNode;
```
Comment (adrien-nf) [inline src/components/Common/SectionTitle/SectionTitle.spec.ts:4]: Watch out for the indentation

File: src/components/Common/SectionTitle/SectionTitle.spec.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+	littleMessage?: string;
```
Comment (adrien-nf) [inline src/components/Common/SectionTitle/SectionTitle.spec.ts:5]: We usually call that one subtitle or something like that

File: src/components/Common/Layout/Layout.tsx (hunk 1 above)
Comment (vbouillennec) [inline src/components/Common/Layout/Layout.tsx:19]: faut dégager ça mdr

**Review by adrien-nf — CHANGES_REQUESTED** [review]

File: src/components/Common/Layout/Layout.tsx [hunk 4]
```diff
@@ -16,7 +16,6 @@ export default function Layout({ children }: LayoutSpec) : JSX.Element {
             items-center
         ">
             <div className="">
```
Comment (adrien-nf) [inline src/components/Common/Layout/Layout.tsx]: Why the empty classname?

**Review by adrien-nf — CHANGES_REQUESTED** [review]

File: src/components/Common/Layout/Layout.tsx (hunk 4 above)
Comment (vbouillennec) [reply src/components/Common/Layout/Layout.tsx]: no id not my fault ! It's like that on the main branch

**Review by adrien-nf — APPROVED** [review]
Perfect!

**Pull request merged**